nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/queue/ui/queue-listing/flowfile-dialog/flowfile-dialog.component.html

```diff
@@ -207,12 +207,7 @@ <h2 mat-dialog-title>FlowFile</h2>
                 <div class="unset neutral-color">Empty string set</div>
             </ng-container>
             <ng-template #nonEmptyValue>
-                <div class="tertiary-color font-medium" *ngIf="title == null; else valueWithTitle">
-                    {{ value }}
-                </div>
-                <ng-template #valueWithTitle>
-                    <div class="tertiary-color font-medium" [title]="title">{{ value }}</div>
-                </ng-template>
+                <div class="tertiary-color font-medium truncate" [title]="value">{{ value }}</div>
```

Comment:
title can be supplied to the outer template `#formatValue` but we are now not using it if supplied. Was that intended? The same comment would apply to the other similar changes made in this PR.

Reply:
ok I see. Let me update this PR.